lib/jekyll/commands/new.rb

```diff
@@ -90,8 +90,11 @@ def create_site(new_blog_path)
             f.write(scaffold_post_content)
           end
 
-          File.open(File.expand_path("Gemfile", new_blog_path), "w") do |f|
-            f.write(gemfile_contents)
+          gemfile_path = File.expand_path("Gemfile", new_blog_path)
+          unless File.exist?(gemfile_path)
```

Comment:
If we're generating a new site, we'll want to write this always. Using --force should indicate some "peril," so I would like us to write no matter what. Fewer support tickets if we know the right gemfile was created :)